src/cloud-sql-instance.ts

```diff
@@ -183,6 +212,12 @@ export class CloudSQLInstance {
   // used to create new connections to a Cloud SQL instance. It throws in
   // case any of the internal steps fails.
   private async performRefresh(): Promise<RefreshResult> {
+    if (this.closed) {
+      // performRefresh() could be delayed by the rate limiter. So
+      // this instance may have been closed.
```

Comment:
Can we update this wording a bit. "So this instance may have been closed" is a bit ambiguous...

Reply:
Fixed.